<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>auth_window</class>
 <widget class="QDialog" name="auth_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLineEdit" name="UserLineEdit"/>
    </item>
    <item row="0" column="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="NameWindow_label">
      <property name="text">
       <string>LOG iN</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="LoginPushButton">
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="PasswordLineEdit"/>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="RegisterPushButton">
      <property name="text">
       <string>Registration</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="exitPushButton">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
